src/main/java/com/telefonica/iot/cygnus/sinks/OrionKafkaSink.java

```diff
@@ -128,16 +118,13 @@ public void start() {
             props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList);
             props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
             props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
-            persistenceBackend = new KafkaProducer<String, String>(props);
+            persistenceBackend = new KafkaBackendImpl(props);
```

Comment:
I would prefer these parameters are passed to KafkaBackendImpl one by one, and if needed, the grouped in a ´Properties` object within KafkaBackendImpl.